config/index.ts

```diff
@@ -241,6 +241,18 @@ export function getConfigDefaultAccount(): string | number | null | undefined {
   return config_DEPRECATED.getConfigDefaultAccount();
 }
 
+export function getDisplayDefaultAccount(
+  isOverride: boolean
```

Comment:
Could you simplify this by removing the `isOverride` option and just calling `getConfigDefaultAccount` when you need the overridden account?